git-hooks/pre-commit

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+import os
+import subprocess
+import sys
+from contextlib import contextmanager
+
+ABORT = 1
+
+@contextmanager
+def cd(new_dir):
+    '''Navigate to a directory in a context and then return to original directory.'''
+    prev_dir = os.getcwd()
+    os.chdir(os.path.expanduser(new_dir))
+    try:
+        yield
+    finally:
+        os.chdir(prev_dir)
+
+if __name__ == '__main__':
+    try:
+        abort = False
+        abort_messages = []
+        skip_flake8 = False
+        FLAKE8_CFG_FILE = '.flake8'
+
+        # Get base folder of repo
+        loc_proc = subprocess.Popen('git rev-parse --show-toplevel'.split(), stdout=subprocess.PIPE)
+        repo_base_folder = loc_proc.communicate()[0].decode('utf-8').strip()
+        flake8_config_path = os.path.join(repo_base_folder, FLAKE8_CFG_FILE)
+
+        # Run pytests
```

Comment:
This code should either be removed or a comment should be added indicated that we intend (if we intend) to add this in the future.